### DISCUSSION TITLE
version compatibility issue here - what should I do ?

### DISCUSSION THREAD

**Discussion by codingchamp1988** [imartinez/privateGPT#1010] [General]
While trying to install private GPT got the below error :

ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
agent-protocol 0.2.4 requires fastapi<0.101.0,>=0.100.0, but you have fastapi 0.99.1 which is incompatible.
botocore 1.29.76 requires urllib3<1.27,>=1.25.4, but you have urllib3 2.0.4 which is incompatible.
google-auth 2.22.0 requires urllib3<2.0, but you have urllib3 2.0.4 which is incompatible.

While trying to solve the above issue it created a new issue : 

Installing collected packages: fastapi
  Attempting uninstall: fastapi
    Found existing installation: fastapi 0.99.1
    Uninstalling fastapi-0.99.1:
      Successfully uninstalled fastapi-0.99.1
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
chromadb 0.4.7 requires fastapi<0.100.0,>=0.95.2, but you have fastapi 0.100.0 which is incompatible.

**Comment by Pichuncho-Sour** [comment]
Don't worry about that errors, you only need to check if the versions indicated on requierements.txt are the same that are installed on python.  to check use the command "pip freeze".  for sure this dependencies are used for other apps and don't interfere on privateGPT installation.
